### ISSUE TITLE
Blade isn't 100%

### ISSUE THREAD

**Issue by jbrooksuk** [github/linguist#3119]
https://gist.github.com/auth/github/callback?browser_session_id=eb0f0dda787683388f10be3c932c0474b3200aa4&code=ffb815687964fd473756&return_to=https%3A%2F%2Fgist.github.com%2Fericbarnes%2F371eac744d09bb06328e878c5f9ac021

Notice how the `@end` tags aren't highlighted properly.

**Comment by jbrooksuk** [comment]
Atom highlights correctly:

![image](https://cloud.githubusercontent.com/assets/246103/16866148/d5a14348-4a61-11e6-8722-60bd0da40838.png)

**Comment by Alhadis** [comment]
It seems to be working fine. The discrepancy is happening because of an inconsistent choice of scope-names. Here's what the grammar's using:

<img src="https://cloud.githubusercontent.com/assets/2346707/16867855/9e09a4d8-4ab7-11e6-8934-3ad96fd7a133.png" width="444" alt="Figure 1" />

For the record, here's how the same code is being rendered with my theme:

<img src="https://cloud.githubusercontent.com/assets/2346707/16867899/e18d0556-4ab7-11e6-916e-3933c696d272.png" width="273" alt="Figure 2" />

Those scope names are essentially just CSS classes, so every syntax theme is going to treat them differently. There're some [guidelines](http://manual.macromates.com/en/language_grammars#naming_conventions) for choosing scope-names, but not every language-grammar will honour them the same way.

**TL;DR:** This is an upstream issue that should be filed at [the repository itself](https://github.com/jawee/language-blade). ;)

**Comment by Alhadis** [comment]
Jesus, I've started closing my replies with `;)` exactly like @pchaigno does. I've clearly spent way too freaking long on this repo.

**Comment by jbrooksuk** [comment]
@Alhadis I always feel like ending replies with `;)` is very suggestive. Like the [Demon Headmaster](https://www.youtube.com/watch?v=4dQyPSJYZxk). I'm off to report it.

**Comment by jbrooksuk** [comment]
If this gets fixed on the upstream, what changes are required on linguist?

**Comment by Alhadis** [comment]
Nothing, mate. Whenever a new release of Linguist is cut, it automatically updates itself with all changes that've been made to the language grammars it uses since the last release.

There's a new release every couple weeks or so. You can test how grammars will look on GitHub using [Lightshow](https://github-lightshow.herokuapp.com/).

;)))))))))))))))))))  
Lisp right up in this bitch.

**Comment by jbrooksuk** [comment]
Lightshow needs updating to include Blade ;)

I'll give it a go.

**Comment by pchaigno** [comment]
> **TL;DR:** This is an upstream issue that should be filed at [the repository itself](https://github.com/jawee/language-blade). ;)

:+1: 

> I've clearly spent way too much freaking time on this repo.

Naah ;)

> Lightshow needs updating to include Blade ;)

Unfortunately, Lightshow is still closed source.
